packages/modules/src/session-validation-modules/ERC20SessionValidationModule.ts

```diff
@@ -0,0 +1,69 @@
+import { defaultAbiCoder } from 'ethers/lib/utils'
+import { ISessionValidationModule } from 'interfaces/ISessionValidationModule'
+import { ERC20SessionKeyData, SessionValidationModuleConfig } from 'utils/Types'
```

Comment:
SessionValidationModuleConfig this is base config?

Reply:
no.. Base config for normal validation modules is different. For session validation modules, each config would be different and specific to SVM.. Couldn't find any common config that we can keep as base for SVM. So kept it separate.